app/src/main/java/mozilla/lockbox/presenter/DisplayItemPresenter.kt

```diff
@@ -89,31 +87,31 @@ class DisplayItemPresenter(
             if (!it.username.isNullOrBlank()) {
                 dispatcher.dispatch(ClipboardAction.CopyUsername(it.username.toString()))
                 dispatcher.dispatch(DataStoreAction.Touch(it.id))
-                view.showToastNotification(R.string.toast_username_copied)
+                dispatcher.dispatch(ToastNotificationAction.ShowCopyUsernameToast)
             }
         }
 
         handleClicks(view.usernameFieldClicks) {
             if (!it.username.isNullOrBlank()) {
                 dispatcher.dispatch(ClipboardAction.CopyUsername(it.username.toString()))
                 dispatcher.dispatch(DataStoreAction.Touch(it.id))
-                view.showToastNotification(R.string.toast_username_copied)
+                dispatcher.dispatch(ToastNotificationAction.ShowCopyUsernameToast)
             }
         }
 
         handleClicks(view.passwordCopyClicks) {
             if (it.password.isNotEmpty()) {
                 dispatcher.dispatch(ClipboardAction.CopyPassword(it.password))
                 dispatcher.dispatch(DataStoreAction.Touch(it.id))
-                view.showToastNotification(R.string.toast_password_copied)
+                dispatcher.dispatch(ToastNotificationAction.ShowCopyPasswordToast)
             }
         }
 
         handleClicks(view.passwordFieldClicks) {
             if (it.password.isNotEmpty()) {
                 dispatcher.dispatch(ClipboardAction.CopyPassword(it.password))
                 dispatcher.dispatch(DataStoreAction.Touch(it.id))
-                view.showToastNotification(R.string.toast_password_copied)
+                dispatcher.dispatch(ToastNotificationAction.ShowCopyPasswordToast)
```

Comment:
At some point, I think we're going to end up with a `vararg` version of `dispatcher.dispatch`. Is this the time to do it?

Reply:
👀

Reply:
I think there is scope to tidy this up, but I don't think we should do it now. Let's just do the simplest thing that works right now.